tox.ini

```diff
@@ -34,6 +34,34 @@ deps =
     # https://github.com/pytest-dev/pytest-rerunfailures/issues/134
     # pytest @ git+https://github.com/pytest-dev/pytest.git
 
+[testenv:packaging]
+description =
+    Do packaging/distribution. If tag is not present or PEP440 compliant upload to
+    PYPI could fail
+# `usedevelop = true` overrides `skip_install` instruction, it's unwanted
+usedevelop = false
+# don't install package in this env
+skip_install = true
+deps =
+    collective.checkdocs >= 0.2
```

Comment:
Isn't this used for `setup.py check` that is not executed here?

Reply:
Added